aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/utils/S3RequestUtils.java

```diff
@@ -23,29 +25,46 @@
  * A utility to help form requests to S3.
  */
 public final class S3RequestUtils {
-    private S3RequestUtils() { }
 
-    @SuppressWarnings("JavadocMethod") // TODO: Add missing documentation
+    @SuppressWarnings("WhitespaceAround") // Looks better this way
+    private S3RequestUtils() {}
```

Comment:
In a separate CR, we should update the checkstyle rules once and for all so that can have empty one-line `{}` bodies on lambdas and constructors